### PR TITLE
Filtering with checkboxes

### PR DESCRIPTION
**Pull request by eivindjahren**
Adds filtering of data using checkboxes for plotly visualisations. Piechart has not gotten filtering as it needs to modify layout.

### CODE REVIEW

File: visualisations/line_chart/examples/line_chart_example.py [hunk 1]
```diff
@@ -10,8 +10,15 @@
 
 line2 = [16, 5, 11, 9]
 
-lines = pd.DataFrame({'line1': line1, 'line2': line2})
+lines = pd.DataFrame({
+    'line 1': line1,
+    'line 2': line2,
+    'line 3': line2,
+    'lin;;; __': line2,
```
Comment (kristofferlund) [inline visualisations/line_chart/examples/line_chart_example.py]: This seems a bit confusing, but there might be a meaning to it?

**Comment by kristofferlund** [comment]
Category boxes are being duplicated, only the ones with the clams have functionality
 screen shot 2018-08-08 at 15 26 03

**Comment by eivindjahren** [comment]
This bug is now fixed.

File: visualisations/line_chart/examples/line_chart_example.py (hunk 1 above)
Comment (eivindjahren) [reply visualisations/line_chart/examples/line_chart_example.py]: Name is now 'line 4' :)

**Pull request merged**